interface-definitions/service_webproxy.xml.in

```diff
@@ -8,6 +8,39 @@
           <priority>500</priority>
         </properties>
         <children>
+          <node name="acl">
+            <properties>
+              <help>Proxy acl settings</help>
```

Comment:
.I believe capitalizing abbreviations makes them easier to read, so this should be "ACL"